app/conf/Configuration.scala

```diff
@@ -227,6 +227,7 @@ class ApplicationConfiguration(val playConfiguration: PlayConfiguration, val isP
 
   object faciatool {
     lazy val breakingNewsFront = "breaking-news"
+    lazy val editEditions = "edit-editions"
```

Comment:
Non-blocking: can I suggest the following?
```suggestion
    lazy val canEditEditions = "edit-editions"
```
`breakingNewsFront` actually refers to the name of a front, as well as being a permission, but `editEditions` is not a front, and it's perhaps not clear this is being used in a permissions context.

Reply:
Yeah sure @jonathonherbert , 
when I started typing this variable I indeed gave "canEditEditions" name but later changed it 😄